### PR TITLE
Remove Backup Version V1

### PR DESCRIPTION
**Pull request by mattl-netflix**
The commits are meant to be reviewed in order to keep the process tractable.

Remove unused CLI tools.
Remove tooling to back up commit logs. It is no longer supported.
Remove IFileSystemContext. It is redundant.
Remove BackupService
Remove BackupVersion.
Remove GCS-specific code. It is no longer in use.
Remove remaining vestiges of backup v1.

### CODE REVIEW

File: priam/src/test/java/com/netflix/priam/backup/TestBackupVerification.java [hunk 1]
```diff
@@ -125,82 +109,23 @@ private void setUp() throws Exception {
...
-                                new DateRange(backupDate + "," + backupDate))
-                        .stream()
-                        .findFirst();
-        Assert.assertTrue(backupMetadata.isPresent());
-        Assert.assertNull(backupMetadata.get().getLastValidated());
+        backupStatusMgr.finish(getBackupMetaData(start, Status.FINISHED));
     }
 
     @Test
     public void verifyBackupVersion1DateRange() throws Exception {
```
Comment (ayushisingh29) [inline priam/src/test/java/com/netflix/priam/backup/TestBackupVerification.java:116]: It will be good if we can update the function name(remove version1) or remove this test completely as v2 test is already present in verifyBackupVersion2DateRange

File: priam/src/main/java/com/netflix/priam/config/PriamConfiguration.java [hunk 2]
```diff
@@ -212,11 +212,6 @@ public String getMaxDirectMemory() {
                 (PRIAM_PRE + ".direct.memory.size.") + instanceInfo.getInstanceType(), "50G");
     }
 
-    @Override
-    public String getBackupCronExpression() {
```
Comment (ayushisingh29) [inline priam/src/main/java/com/netflix/priam/config/PriamConfiguration.java:216]: Why is this removed?

**Review by chengw-netflix — APPROVED** [review]

File: priam/src/main/java/com/netflix/priam/config/PriamConfiguration.java (hunk 2 above)
Comment (mattl-netflix) [reply priam/src/main/java/com/netflix/priam/config/PriamConfiguration.java:216]: It was only referenced in SnapshotBackup which was removed in this cleanup. The modern equivalent is getSnapshotMetaServiceCronExpression() used in SnapshotMetaTask.

File: priam/src/test/java/com/netflix/priam/backup/TestBackupVerification.java (hunk 1 above)
Comment (mattl-netflix) [reply priam/src/test/java/com/netflix/priam/backup/TestBackupVerification.java:116]: Agreed. The test is redundant and should be removed. I will do that in a follow-up.
